azurerm/resource_arm_dashboard.go

```diff
@@ -0,0 +1,153 @@
+package azurerm
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/services/preview/portal/mgmt/2019-01-01-preview/portal"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/response"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/tags"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+	"regexp"
+)
+
+func resourceArmDashboard() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmDashboardCreateUpdate,
+		Read:   resourceArmDashboardRead,
+		Update: resourceArmDashboardCreateUpdate,
+		Delete: resourceArmDashboardDelete,
```

Comment:
could we add import support here? that should be:

```suggestion
		Delete: resourceArmDashboardDelete,
		Importer: &schema.ResourceImporter{
			State: schema.ImportStatePassthrough,
		},
```

Reply:
actually understand what import does now :) - it's added